packages/shared/tailwind.config.js

```diff
@@ -127,14 +127,15 @@ module.exports = {
       1: '1',
       2: '2',
       3: '3',
-      rank: '3',
-      header: '4',
-      postNavigation: '5',
-      sidebar: '7',
-      tooltip: '8',
-      popup: '9',
-      modal: '10',
-      max: '100',
+      rank: '30',
+      header: '75',
+      postNavigation: '76',
+      sidebar: '70',
+      tooltip: '80',
+      popup: '90',
+      modal: '100',
+      cookie: '2000',
+      max: '1000',
```

Comment:
Max is lower then cookie?

Reply:
It was before as well, it was `z-[200]` vs z-100 that was previous max.

Reply:
Last cookie value comes from here https://github.com/dailydotdev/apps/pull/1936. From What I see `z-max` should be ok as well so we can change to that.

Reply:
Hmm yeah let's maybe try that and see if it works?

Reply:
I tested already locally, it works, only things that have the z-max as well and could interfeer with banner are:
- toast
- context menu 

since those are temp and parts of user interaction it makes sense they would be on the same level, it is not like they obscure premaritally.